README.md

```diff
@@ -12,6 +12,7 @@ torchaudio: an audio library for PyTorch
 
 Dependencies
 ------------
+* vesis84/kaldi-io-for-python commit cb46cb1f44318a5d04d4941cf39084c5b021241e or above
```

Comment:
mark this as Optional dependency. 
So move it to the bottom of the list, and prefix it with `[optional] `